aicsimageio/aics_image.py

```diff
@@ -86,25 +118,36 @@ def __init__(
         -----
         Constructor for AICSImage class intended for providing a unified interface for
         dealing with microscopy images. To extend support to a new reader simply add a
-        new reader child class of Reader ([readers/reader.py]) and add the class to
+        new reader child class of Reader ([readers/reader.py]) and add the class to the
         SUPPORTED_READERS variable.
         """
-        pass
-
-    @staticmethod
-    def determine_reader(image: types.ImageLike) -> Reader:
-        """
-        Returns
-        -------
-        reader: Reader
-            Cheaply check to see if a given file is a recognized type and return the
-            appropriate reader for the file.
-
-        Raises
-        ------
-        TypeError: Unsupported file format.
-        """
-        pass
+        # Check known dims
+        if known_dims is not None:
+            if not all(
+                [
+                    d in dimensions.DEFAULT_DIMENSIONS_ORDER_LIST_WITH_SAMPLES
+                    for d in known_dims
+                ]
+            ):
+                raise exceptions.InvalidDimensionOrderingError(
+                    f"The provided dimension string to the 'known_dims' argument "
+                    f"includes dimensions that AICSImage does not support. "
+                    f"Received: '{known_dims}'. "
+                    f"Supported dimensions: "
+                    f"{dimensions.DEFAULT_DIMENSIONS_ORDER_LIST_WITH_SAMPLES}."
+                )
```

Comment:
Much easier to add tests for this with `ArrayLikeReader` implented. Holding off until that is in.